test/utils/identity-resolution.app.factory.ts

```diff
@@ -0,0 +1,46 @@
+import { INestApplication, ModuleMetadata, ValidationPipe } from '@nestjs/common';
```

Comment:
a lot of code tat is then needs to be maintained, to discuss

Reply:
When we roll out mutation and then business logic, a lot of code has to go. This in on my special watch-list. Final result of transformation must be less code than before, not more :) And it should work faster. Otherwise it is bad transformation.